src/components/layout/marketing-header.css

```
@@ -0,0 +1,47 @@
+.lp {
+  --lp-border-strong: 0 0% 80%;
+  --lp-shadow-surface: 0 30px 60px -30px rgba(9, 9, 11, 0.25);
+  font-feature-settings: "cv11", "ss01", "ss03";
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)